src/worker.rs

```diff
@@ -79,7 +80,20 @@ fn list_files(logger: &Logger, name: &str, recursive: bool) -> Result<Vec<File>,
             let mut in_file_entry = false;
             let (mut in_name, mut in_author, mut in_date, mut in_size) = (false, false, false, false);
             let (mut name, mut author, mut date, mut size, mut revision) = (None, None, None, None, None);
-
+            let sha_output = Command::new("svn")
+                .arg("propget")
+                .arg("git-commit")
+                .args("--revprop")
+                .args("-r")
+                .args("HEAD")
+                .arg(format!("{}/{}", ORGANIZATION_ROOT, name))
+                .output();
+            let mut sha = match sha_output {
```

Comment:
If this doesn't work, we should be returning an `Err`.

Reply:
Focused on getting the Ok case to work for now, but I'll implement it once the rest works.